includes/class-wc-payment-checker-maksuturva.php

```diff
@@ -52,18 +52,43 @@ class WC_Payment_Checker_Maksuturva {
 	public static function install_db() {
 		global $wpdb;
 
+		// Truncate the table if the current version is below the specified DB version.
+		$version = get_option( WC_Maksuturva::OPTION_DB_VERSION );
+		if ( version_compare( $version, WC_Maksuturva::DB_VERSION ) < 0 ) {
+			self::truncate_table();
+		}
+
 		$tbl = $wpdb->prefix . self::TABLE_NAME;
 
 		$sql = 'CREATE TABLE `' . $tbl . '` (
 		`payment_id` varchar(36) NOT NULL,
```

Comment:
Can the payment_id be unique after the truncate? does dbDelta handle the change?